packages/medusa/src/services/claim-item.ts

```diff
@@ -70,6 +70,13 @@ class ClaimItemService extends TransactionBaseService {
         .withTransaction(manager)
         .retrieve(item_id)
 
+      if (!lineItem.variant_id) {
+        throw new MedusaError(
+          MedusaError.Types.NOT_ALLOWED,
+          "Cannot claim a virtual line item"
```

Comment:
question: Is virtual the correct word to use here? Would maybe choose `custom` or `Cannot claim line item that has no variant`